pkg/service/project.go

```diff
@@ -60,8 +61,13 @@ func (s *projectService) Create(ctx context.Context, project *systemv3.Project)
 		return nil, fmt.Errorf("missing organization in metadata")
 	}
 
+	err, matched := MatchStringToRegx(project.Metadata.GetName(), common.PROJECT_NAME_REGEX)
+	if err != nil || !matched {
+		return nil, fmt.Errorf("cluster name contains invalid characters. valid characters are `%s`",common.PROJECT_NAME_REGEX)
```

Comment:
can the error message be more user friendly ? It would not make showing a regex expression in the error message for non tech related users. 

It can be something like "Valid characters are a-z0-9 and hyphen, except at the beginning or the end"

Reply:
Or `Valid characters are alphanumeric and hyphen, except at the beginning or the end`

Reply:
Agree will make the message more user-friendly.